test/data/images-ref/centos/9/x86_64/ami/centos_9-x86_64-ami-empty.json

```diff
@@ -196,6 +196,96 @@
                 },
                 {
                   "id": "sha256:90aebae315675cbf04612de4f7d5874850f48e0b8dd82becbeaa47ca93f5ebfb"
+                },
```

Comment:
Can this maybe be a separate PR or commit? The existing commit says: `ami for aarch64` but this change seems unrelated to the commit description

Reply:
Yes, I think this was accidental. Let me verify.